deployment/docker/Dockerfile

```diff
@@ -10,6 +10,11 @@ RUN apt-get update -y; apt-get -y --force-yes install yui-compressor gettext
 RUN wget https://github.com/jgm/pandoc/releases/download/1.17.1/pandoc-1.17.1-2-amd64.deb
 RUN dpkg -i pandoc-1.17.1-2-amd64.deb && rm pandoc-1.17.1-2-amd64.deb
 
+# Added because of issue with building cryptography.io using pip
+# This flag disabled rust build, but only for this particular version
+# In the future, we may have to include rust toolchain in the Dockerfile
+ENV CRYPTOGRAPHY_DONT_BUILD_RUST=1
+
```

Comment:
The actual fix is just these lines.

The other files are modified to make my dev env tidy in GitHub Codespace (unrelated with the issue)